cloud-benchmark/src/main/resources/logback.xml

```diff
@@ -9,4 +9,6 @@
   </root>
   <logger name="xtdb" level="INFO" />
   <logger name="com.azure" level="ERROR" />
+  <logger name="reactor" level="ERROR" />
+  <logger name="reactor.netty" level="ERROR" />
```

Comment:
shouldn't need `reactor.netty` here - Java logging forms a hierarchy, so if `reactor.netty` isn't specified it defaults to `reactor`